Formula/doctl.rb

```diff
@@ -23,11 +23,18 @@ def install
     ln_sf buildpath, buildpath/"src/github.com/digitalocean/doctl"
     Language::Go.stage_deps resources, buildpath/"src"
 
-    system "go", "build", "-ldflags", "-X github.com/digitalocean/doctl/Build=#{version}", "github.com/digitalocean/doctl/cmd/doctl"
+    doctl_version = version.to_s.split(/\./)
+    base_flag = "-X github.com/digitalocean/doctl"
+    ldflags = "#{base_flag}.Major=#{doctl_version[0]}"
```

Comment:
As-is this is getting passed as one argument to `system`.  We can fix that and make it more readable like this:
```ruby
args = %W[
  #{base_flag}.Major=#{doctl_version[0]}
  #{base_flag}.Minor=#{doctl_version[1]}
  #{base_flag}.Patch=#{doctl_version[2]}
  #{base_flag}.Label=release
  github.com/digitalocean/doctl/cmd/doctl
]
system "go", "build", "-ldflags", *args